code/game/objects/items/tanks/jetpack.dm

```diff
@@ -17,6 +17,7 @@
 /obj/item/tank/jetpack/Initialize()
 	. = ..()
 	ion_trail = new
+	ion_trail.auto_process = FALSE

```

Comment:
This looks unnecessary, you're doing everything move react.

Reply:
For what it need in autoprocessing when i manually call on demand?

https://github.com/tgstation/tgstation/blob/8358169b8e5a37508ae4f15ac1db007ef14a4a88/code/game/objects/effects/effect_system/effects_other.dm#L34-L41